### DISCUSSION TITLE
kyuubi access S3 data with Spark

### DISCUSSION THREAD

**Discussion by changqing** [apache/kyuubi#4511] [Q&A]
Hi,
Our Kyuubi is running on K8s and already have aws jars in the image。 Is there other ways to load the S3 related jars？

![image](https://user-images.githubusercontent.com/2008743/224990258-2152299f-ed60-4e43-8b55-2871e3132e98.png)

thansk

**Comment by changqing** [comment]
我的版本是1.7.0

**Reply by pan3793** [reply]
You need to add the following 4 jars into `SPARK_HOME/jars`:
```
spark-hadoop-cloud_2.12
hadoop-cloud-storage
hadoop-aws
aws-java-sdk-bundle
```

**Reply by pan3793** [reply]
it may be helpful
https://github.com/apache/kyuubi/tree/master/docker/playground
